projects/coordinator/src/main/java/org/batfish/coordinator/Main.java

```diff
@@ -171,16 +171,51 @@ private static void startWorkManagerService(
     }
   }
 
+  private static void startWorkManagerServiceV2(
```

Comment:
Cloning 35 lines of code is likely not the right answer.

The existing function was configured to be usable by both v1 and v2 (that's why it takes serviceClass and jsonFeature as arguments). Keep that pattern going if possible.